src/main/resources/assets/trinkets/lang/pl_pl.json

```diff
@@ -0,0 +1,11 @@
+{
+	"trinkets.slot.head.mask": "Maska",
+	"trinkets.slot.chest.necklace": "Naszyjnik",
+	"trinkets.slot.chest.backpack": "Plecak",
+	"trinkets.slot.chest.cape": "Peleryna",
+	"trinkets.slot.legs.belt": "Pas",
+	"trinkets.slot.feet.aglet": "Aglet",
```

Comment:
Not entirely sure if this or `skuwka` is better? I mean, this version is surely well known thanks to a certain Disney cartoon...

Reply:
I was going to change this but did a little research and found this:

<details>

Nie jest to jednoznaczne ze skuwką po angielsku. W polskim języku słowo "skuwka" dla poprawności powinno być uzupełnione; "skuwka pióra", skuwka sznurowadła". Po angielsku odpowiednikiem skuwki jest "ferrule" (a nie aglet). Skuwka, jak sama nazwa po polsku głosi, to coś skute, zakute (najczęściej metalowe), więc te już zatapiane nie są skuwkami, a właśnie agletami. Natomiast zakute też są agletami, bo to nazwa nie z angielskiego, a z łaciny (od słowa igła) i tak też były w przeszłosci po polsku nazywane. Jak najbardziej poprawnym określeniem końcówki sznurowadła jest więc AGLET po polsku. Kiedyś Polacy mieli bogatszy język, ale dzisiaj, mają większe przekonanie o swojej wiedzi, jezęli więcej korzystają z internetu :) - https://sjp.pl/aglet
</details>

Now I'm not sure so looks like we should wait for one more person to decide :D

Reply:
Oh, you convinced me now. I did this during a short break at work without any research :P